src/Directory.Build.props

```diff
@@ -1,5 +1,404 @@
 <Project>
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
+
```

Comment:
Would have been nice to split meaningful changes into a separate commit to make this more reviewable. 😕

Reply:
Yeah I know, I'm sorry for that. A lot of changes were connected and I wasn't sure if it will work out like I imagined. I will try to avoid that in future.

Reply:
I tend to do rebasing and such to set up nice PRs for my larger sets of changes when the number of commits and backtracking gets too big to submit as-is.